bin/tachyon-start.sh

```diff
@@ -28,6 +28,8 @@ MOPT is one of:
 
 -N  Do not try to kill prior running masters and/or workers in all or local
 
+-w  Wait for processes to end before returning
```

Comment:
I think -N has comment "Do not ...", starting with capitalized "D", so may be fine